Kernel_23/doc/Kernel_23/CGAL/Weighted_point_2.h

```diff
@@ -47,11 +47,8 @@ class Weighted_point_2
 
   /*!
   introduces a weighted point from point `p` and weight `0`.
-
-\warning The `explicit` keyword is used to not have accidental implicit conversions
-         between Point_2 and Weighted_point_2.
   */
-  explicit Weighted_point_2(const Point_2<Kernel>& p);
```

Comment:
To avoid mixing branch purposes. This is a fix to the initial implementation of Weighted_points. Removing implicit conversions is done in https://github.com/afabri/cgal/tree/Kernel_Weighted_point_without_conversion-GF (in particular, https://github.com/afabri/cgal/commit/0e7df2413cc46fa91a71ec4930ef9f30dd1b8c32 )